notes/inheritance.js

```diff
@@ -0,0 +1,135 @@
+/**
+  Let's say we have a class Monster. This class will define all of our Monsters at a generic level.
+  It will hold attributes that each Monster, be it a Zombie, a Skeleton, a Banshee, etc should have.
+**/
+class Monster {
+  constructor(name, hitpoints, level, inventory, nickname) {
+    this.name = name;
+    this.hitpoints = hitpoints;
+    this.level = level;
+    this.inventory = inventory || [];
```

Comment:
You could potentially just default this in the constructor parens. ```constructor(inventory = []) ```
which would allow you to forego the ||.